terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -18,6 +18,7 @@ landing_page_url = "/register-an-overseas-entity/sold-land-filter?start=0"
 landing_page_starting_new_url = "/register-an-overseas-entity/starting-new"
 payment_fee = "100"
 update_landing_page_url = "/update-an-overseas-entity/continue-with-saved-filing"
+remove_landing_page_url = "/gov-uk-remove-guidance-page"
```

Comment:
Will it be the flag or the url used when deciding whether to an update page or a remove page? e.g. the fork from the "Who can we contact about this filing page?"

Reply:
It's the URL parameter, which is ultimately part of the URL. Currently, if not present then it's the update journey, if it is present AND its value is 'remove' then it's the remove journey.

Reply:
So we keep open the possibility of introducing a parameter value for update by choosing to use a value but will not do now, which means this functions like a flag base don presence or absance.

Reply:
Nearly. So absence of the parameter essentially indicates the update journey (for now, as no code is checking this) but presence (alone) is not enough - the value must be `remove` for the app to recognise it as the remove journey.